app/src/androidTest/java/com/karumi/ui/view/SuperHeroDetailActivityTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.karumi.ui.view
+
+import android.os.Bundle
+import com.github.salomonbrys.kodein.Kodein.Module
+import com.github.salomonbrys.kodein.bind
+import com.github.salomonbrys.kodein.instance
+import com.karumi.data.repository.SuperHeroRepository
+import com.karumi.domain.model.SuperHero
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+
+class SuperHeroDetailActivityTest : AcceptanceTest<SuperHeroDetailActivity>(
+    SuperHeroDetailActivity::class.java) {
+
+    @Mock private lateinit var repository: SuperHeroRepository
+
+    @Test
+    fun showsAvengersBadgeIfSuperHeroIsPartOfTheAvengersTeam() {
+        val superHero = givenAnAvenger()
+
+        val activity = startActivity(superHero)
+
+        compareScreenshot(activity)
+    }
+
+    @Test
+    fun doesNotShowAvengersBadgeIfSuperHeroIsNotPartOfTheAvengersTeam() {
+        val superHero = givenThereIsASuperHero(false)
+
+        val activity = startActivity(superHero)
+
+        compareScreenshot(activity)
+    }
+
+    private fun givenAnAvenger(): SuperHero {
+        return givenThereIsASuperHero(true)
+    }
+
+    private fun givenThereIsASuperHero(isAvenger: Boolean): SuperHero {
+        val superHeroName = "SuperHero"
+        val superHeroDescription = "Super Hero Description"
+        val superHero = SuperHero(superHeroName, null, isAvenger, superHeroDescription)
+        `when`(repository.getByName(superHeroName)).thenReturn(superHero)
```

Comment:
Can we add an extension to Mockito so that we don't need to add the backquotes every single time we mock something? `whenever`, `on` or any other similar name would do the trick IMO